### PR TITLE
Fix layout

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: samples/python/visual_language_chat/visual_language_chat.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def read_image(path: str) -> Tensor:
 
     '''
     pic = Image.open(path).convert("RGB")
-    image_data = np.array(pic.getdata()).reshape(1, 3, pic.size[1], pic.size[0]).astype(np.byte)
+    image_data = np.array(pic.getdata()).reshape(1, pic.size[1], pic.size[0], 3).astype(np.byte)
     return Tensor(image_data)
```
Comment (ilya-lavrenov) [inline samples/python/visual_language_chat/visual_language_chat.py:40]: @akuporos do you think OpenVINIO Python can support ov.Tensor creation from Pillow image directly ?

File: src/cpp/src/visual_language/input_embedder.cpp [hunk 2]
```diff
@@ -764,11 +764,6 @@ class InputsEmbedderLLaVANext : public InputsEmbedderLLaVA {
         size_t patch_seq_len = shape[1];
         size_t embed_dim = shape[2];
 
-        OPENVINO_ASSERT(
-            num_patches == num_patch_height * num_patch_width,
-            "Number of patches does not match the specified grid size"
-        );
-
```
Comment (yatarkan) [inline src/cpp/src/visual_language/input_embedder.cpp:771]: Why this assert is removed?

Comment (Wovchena) [reply src/cpp/src/visual_language/input_embedder.cpp:771]: It was triggered and I was hoping that's something that isn't needed :)

File: samples/python/visual_language_chat/visual_language_chat.py (hunk 1 above)
Comment (akuporos) [reply samples/python/visual_language_chat/visual_language_chat.py:40]: yes, we can do it

Comment (akuporos) [reply samples/python/visual_language_chat/visual_language_chat.py:40]: ticket #156053

**Review by ilya-lavrenov — APPROVED** [review]

**Review by yatarkan — APPROVED** [review]

**Pull request merged**